homepage/homepage.css

```diff
@@ -0,0 +1,113 @@
+:root {
+  --primary-color: #bf3325;
+  --secondary-color: #ef5354;
+  --offwhite-color: #ffff;
+  --background-color: #ff6263;
+}
+body {
+  font-family: 'Poppins', sans-serif;
+  font-family: 'Roboto', sans-serif;
+}
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+  padding: 20px;
+}
+
+nav {
+  flex: 1;
+  text-align: right;
+}
+
+nav ul img {
+  width: 30px;
+  height: 25px;
+}
+
+nav ul {
+  display: inline-block;
+  list-style-type: none;
+}
+nav ul li {
+  display: inline-block;
+  margin-right: 20px;
+}
+a {
+  text-decoration: none;
+  color: var(--offwhite-color);
+  font-size: 20px;
+  font-weight: 550;
+}
+
+.container {
+  max-width: 1300px;
+  margin: auto;
+  padding: 0 25px;
+}
+.row {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+.col-2 {
+  flex-basis: 50%;
+  min-width: 300px;
+}
+
+.col-2 img {
+  max-width: 100%;
+  padding: 50px 0;
+}
+
+.col-2 h1 {
+  font-size: 50px;
+  line-height: 60px;
+  margin: 25px 0;
+}
+.btn {
+  display: inline-block;
+  background-color: var(--primary-color);
+  color: var(--offwhite-color);
+  margin: 30px 0;
+  padding: 8px 30px;
+  border-radius: 30px;
+  transition: background 0.5s;
+}
+
+.btn:hover {
+  background-color: var(--secondary-color);
+}
+.header {
+  background: radial-gradient(var(--offwhite-color), var(--primary-color));
+}
+.header .row {
+  margin-top: 70px;
+}
+.categories {
+  margin: 60px 0;
+}
+.col-3 {
+  flex-basis: 30%;
+  min-width: 250px;
```

Comment:
try using relative units here so that it will scale when a user visits the site on larger screens.

Reply:
thank you will implement